.github/workflows/deploy.yml

```
@@ -0,0 +1,45 @@
+#
+# Build and deploy the website
+#
+name: Deploy
+
+# Run on main branch only
+on:
+  pull_request: # enable pull_request for testing
+  push:
+    branches:
+      - master
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Cancel Previous Runs
+        uses: styfle/cancel-workflow-action@0.9.0
+
+      - name: Checkout
+        uses: actions/checkout@v2.3.4
+        with: 
+          submodules: 'true'
+
+      - name: Setup Hugo
+        uses: peaceiris/actions-hugo@v2
+        with:
+          hugo-version: 0.61.0
```

Let's try if the latest hugo version works
```suggestion
          hugo-version: 0.83.1
```